test/e2e/e2e_helpers_test.go

```diff
@@ -424,6 +422,59 @@ func sortNamespacedNames(nns []types.NamespacedName) {
 }
 
 func testDataPath(description ginkgo.GinkgoTestDescription) string {
-       path := filepath.Join(description.ComponentTexts...)
-       return strings.ReplaceAll(path, " ", "_")
+	path := filepath.Join(description.ComponentTexts...)
+	return strings.ReplaceAll(path, " ", "_")
+}
+
+func GetIstioObject(istio *istiov1beta1.Istio, namespace, name string) error {
```

Comment:
Could this be an unexported function maybe?
```suggestion
func getIstioObject(istio *istiov1beta1.Istio, namespace, name string) error {
```

Reply:
This method is used inside the test e2e_test.go, so it needs to be exported.

Reply:
As far as I know, since this function is used from the same package you don't need to export it.

Reply:
Fixed.
I am new to go, so I thought package == file.

Reply:
No problem, thanks for the fix!